### DISCUSSION TITLE
question: aapt or aapt2 used for building

### DISCUSSION THREAD

**Discussion by MondayNitro** [iBotPeaches/Apktool#3464] [Q&A]
it might be a dumb or even wrong question, but what does apktool use for building by default? i saw apktool has aapt, aapt64, aapt2, aapt2_64. im coming from apktool m, where it asks to use aapt or aapt2, so i couldnt help but draw parallels between the two.
also, is it possible to set default a particular tool for building?

**Comment by iBotPeaches** [comment]
aapt2 became default in Apktool 2.9.0.

You can't really set a default, but you can use `--use-aapt2` and `--use-aapt1` during [building](https://apktool.org/docs/cli-parameters#building-options)
